accounts/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from .models import User, Organization, Admin, Agent, Customer
+
+# Register your models here.
+admin.site.register(User)
+admin.site.register(Organization)
+admin.site.register(Admin)
+admin.site.register(Agent)
+admin.site.register(Customer)
```

Comment:
Do not use dot imports. Write the name of the application with it e.g 

`from accounts.models import (
    Admin,
    Agent,
    Customer,
    Organization,
    User)`

Also import alphabetically and leave a line after type of imports. e.g

<internal imports>

<third party imports>

<application imports>

An example would be 

`import time

from django.contrib import admin

from accounts.models import User
`